modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIAdminImpl.java

```diff
@@ -435,6 +466,23 @@ public SettingsDTO getSettings() throws ApiException {
                 .throttlingDenyPoliciesPostWithHttpInfo(Constants.APPLICATION_JSON, denyPolicyDTO);
     }
 
+    /**
+     * Updates an deny throttling policy.
+     *
+     * @param conditionId policy id of the deny throttling policy to be updated.
+     * @param conditionType condition type of the deny throttling policy to be.
+     * @param blockingConditionStatusDTO deny throttling policy status DTO to be updated.
+     * @return API response returned by API call.
+     * @throws ApiException if an error occurs while creating the deny throttling policy.
+     */
+    public ApiResponse<BlockingConditionDTO> updateDenyThrottlingPolicy(String conditionId, String conditionType, BlockingConditionStatusDTO blockingConditionStatusDTO) throws ApiException {
```

Comment:
Looks line in here line limit(120) exceeds?

Reply:
Refactored